designer/client/src/components/tips/error/ErrorTips.tsx

```diff
@@ -1,29 +1,46 @@
-import React from "react";
-import { v4 as uuid4 } from "uuid";
-import { isEmpty } from "lodash";
-import { NodeValidationError } from "../../../types";
+import React, { useMemo } from "react";
+import { concat, isEmpty } from "lodash";
 import { Props } from "./Errors";
-import { NodeErrorsTips } from "./NodeErrorsTips";
+import { NodeValidationError } from "../../../types";
+import { v4 as uuid4 } from "uuid";
+import NodeErrorsLinkSection from "./NodeErrorsLinkSection";
+import i18next from "i18next";
 
 export const ErrorTips = ({ errors, showDetails, scenario }: Props) => {
-    const globalErrors = errors.globalErrors;
-    const nodeErrors = errors.invalidNodes;
-    const propertiesErrors = errors.processPropertiesErrors;
+    const { globalErrors, processPropertiesErrors, invalidNodes } = errors;
 
-    const globalErrorsTips = (globalErrors: NodeValidationError[]) => <div>{globalErrors.map((error) => globalError(error, null))}</div>;
+    const invalidNodeIds = Object.keys(invalidNodes);
 
-    const globalError = (error: NodeValidationError, suffix) => (
+    const renderGlobalErrorSimpleMessage = (error: NodeValidationError) => (
         <span key={uuid4()} title={error.description}>
```

Comment:
We cannot use uuid4() ad a key, because it will force re-render of the component on every react pre-render. We need to use something stable as a key, which won't change during the lifecycle of the component.

Reply:
Thanks, havent thought about it. I changed it to use index. I also removed displaying fieldname since for global errors its always null and removed the function.